bootstrap-v4.sh

```diff
@@ -36,7 +36,8 @@ function set_internal_variables {
     ACTIVITY_LOG=$INSTALL_BASE/shibboleth-idp4-installer/activity.log
 
     GIT_REPO=https://github.com/ausaccessfed/shibboleth-idp4-installer.git
-    GIT_BRANCH=master
+#    GIT_BRANCH=master
+    GIT_BRANCH=develop
```

Comment:
will this be changed back to `master`?

Reply:
Yes, proir to getting merged into master.